bedrock/firefox/templates/firefox/family/index.html

```diff
@@ -25,9 +25,6 @@
   {% endif %}
 {% endblock %}
 
-{% block tabzilla_css %}{% endblock %}
-{% block tabzilla_js %}{% endblock %}
-
```

Comment:
Great @jpetto! I have added another note in bug 1186605 regarding possible removal of template-resp.css.